libs/platform/ebpf_epoch.c

```diff
@@ -445,7 +489,22 @@ _ebpf_epoch_get_release_epoch(_Out_ int64_t* release_epoch)
         lock_state = ebpf_lock_lock(&_ebpf_epoch_cpu_table[cpu_id].lock);
 
         // Clear the flush timer flag.
-        _ebpf_epoch_cpu_table[cpu_id].flags.timer_armed = false;
+        _ebpf_set_per_cpu_flag(&_ebpf_epoch_cpu_table[cpu_id], EBPF_EPOCH_PER_CPU_TIMER_ARMED, false);
+
+        if (!ebpf_list_is_empty(&_ebpf_epoch_cpu_table[cpu_id].free_list)) {
+            if (_ebpf_get_per_cpu_flag(&_ebpf_epoch_cpu_table[cpu_id], EBPF_EPOCH_PER_CPU_STALE)) {
+                // Schedule DPC
+                if (!_ebpf_epoch_cpu_table[cpu_id].stale_worker) {
+                    if (ebpf_allocate_non_preemptible_work_item(
+                            &_ebpf_epoch_cpu_table[cpu_id].stale_worker, cpu_id, _ebpf_epoch_stale_worker, NULL) ==
+                        EBPF_SUCCESS) {
+                        ebpf_queue_non_preemptible_work_item(_ebpf_epoch_cpu_table[cpu_id].stale_worker, NULL);
```

Comment:
Bug:
Only queues it once after it's created. If it already exists, it doesn't queue it.